hieradata/clients/controller.trusted.ci.jenkins.io.yaml

```
@@ -121,6 +121,27 @@ profile::jenkinscontroller::jcasc:
           virtualNetworkName: trusted-vnet
           virtualNetworkResourceGroupName: trusted
           subnetName: trusted-vnet-trusted-jenkins-ci-io-ephemeral-agents
+        - name: "win-2022" # The name must not contains "windows" or Azure API complains :facepalm:
+          description: "Windows 2022"
+          imageDefinition: jenkins-agent-windows-2022-amd64
+          os: "windows"
+          os_version: "2022"
+          launcher: "inbound"
```

```suggestion
          launcher: "ssh"
```

Inbound is not possible for now on trusted.ci.jenkins.io, because of https://github.com/jenkinsci/azure-vm-agents-plugin/issues/422